plugins/circleci/test/circleci-config.test.ts

```diff
@@ -191,5 +191,18 @@ describe('CircleCI config hook', () => {
       }
       expect(jobs[3]).toEqual(expect.objectContaining(partialExpectedJob))
     })
+
+    it('should match the snapshot', async () => {
```

Comment:
**suggestion:** We should rephrase this test to say they purpose of the snapshot.